test/connectors/ChargeHistoryConnectorSpec.scala

```diff
@@ -18,29 +18,31 @@ package connectors
 
 import audit.mocks.MockAuditingService
 import config.FrontendAppConfig
-import mocks.MockHttp
-import models.chargeHistory.{ChargeHistoryResponseModel, ChargesHistoryErrorModel}
+import mocks.{MockHttp, MockHttpV2}
+import models.chargeHistory.ChargesHistoryResponse.ChargesHistoryResponse
+import models.chargeHistory.{ChargeHistoryModel, ChargeHistoryResponseModel, ChargesHistoryErrorModel, ChargesHistoryModel}
 import play.api.Configuration
 import play.mvc.Http.Status
 import testConstants.BaseTestConstants.{chargeReference, testMtditid}
-import testConstants.ChargeHistoryTestConstants.{testChargeHistoryErrorModelParsing, testInvalidChargeHistoryDetailsModelJson, testValidChargeHistoryDetailsModelJson, testValidChargeHistoryModel}
+import testConstants.ChargeHistoryTestConstants.{testChargeHistoryErrorModelParsing, testValidChargeHistoryModel}
 import testUtils.TestSupport
-import uk.gov.hmrc.http.HttpResponse
 import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
 
+import java.time.LocalDate
 import scala.concurrent.Future
 
-class ChargeHistoryConnectorSpec extends TestSupport with MockHttp with MockAuditingService {
+class ChargeHistoryConnectorSpec extends TestSupport with MockHttp with MockHttpV2 with MockAuditingService {
```

Comment:
I think we don't need MockHttp to be exposed in the scope of this spec? as we can use only one or anther ?